jenkins-ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent any
+    stages {
+        stage('all-in-one: clean environment') { 
+            steps { 
+                sh 'oc get all'
+                sh 'oc delete all,template,daemonset -l jaeger-infra'
```

Comment:
Add `configmap` as well, and possibly other objects that we don't actually make use of today. We have a PR opened that adds `configmap` to the mix, so, adding all possible objects here would make it easier for similar future additions.